docs/reference/composition.md

```diff
@@ -734,6 +734,12 @@ Currently only `multiply` is supported.
      type: Convert
      convert: ToJson
 
+# The output will be the hash of the JSON representation of the 'from' field.
+- type: string
+  string:
+     type: Convert
+     convert: ToSha1 # alternatives: 'ToSha256' or 'ToSha512'
```

Comment:
Please update the documentation under https://github.com/crossplane/docs and share the PR link here.

Reply:
Here you go: https://github.com/crossplane/crossplane/pull/3498

Reply:
The link points to the PR itself?

Reply:
Nm, I just found, already merged https://github.com/crossplane/docs/pull/293